src/ray/core_worker/transport/direct_actor_transport.cc

```diff
@@ -84,7 +81,7 @@ void CoreWorkerDirectActorTaskSubmitter::HandleActorUpdate(
     // Check if this actor is the one that we're interested, if we already have
```

Comment:
Not part of this PR explicitly but this comment is very confusing. Can you clarify it please?

Reply:
It is unintuitive to me that we would create a new connection if we have an existing one.

Reply:
Pending requests means we don't have any existing connection.